sconstruct.log

```diff
@@ -1,4 +1,309 @@
+  1.     import delimited "output/data/data.txt", delimiter("|") varnames(nonam
+> es) clear
+  2.     rename v1 obs_value
+  3.     gen obs_number = _n
+  4.     graph twoway line obs_value obs_number
+  5.     graph export "output/analysis/plot.eps"
+  6. end
+
+. 
+. main    
+(1 var, 201 obs)
+(file output/analysis/plot.eps written in EPS format)
+
+. 
+end of do-file
+
+*** Builder log created:    2017-01-19 10:51:03
+*** Builder log completed:  2017-01-19 10:51:04
+ 
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./online_appendix.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./online_appendix.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+
+*** Builder log created:    2017-01-19 10:51:04
+*** Builder log completed:  2017-01-19 10:51:05
+ 
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./paper.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./paper.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+
+*** Builder log created:    2017-01-19 10:51:05
+*** Builder log completed:  2017-01-19 10:51:07
+ 
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./slides.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016) (preloaded format=pdflatex)
+ restricted \write18 enabled.
+entering extended mode
+(./slides.tex
+LaTeX2e <2016/03/31>
+Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
+
+
+^^^ New build: 2017-01-19 10:51:02 ^^^
+scons: done reading SConscript files.
+scons: Building targets ...
+build_r(["output/data/data.txt"], ["source/data/build.R"])
+build_stata(["output/analysis/plot.eps"], ["source/analysis/analysis.do"])
+build_lyx(["output/paper/online_appendix.pdf"], ["source/paper/online_appendix.lyx"])
+build_lyx(["output/paper/paper.pdf"], ["source/paper/paper.lyx"])
+build_lyx(["output/talk/slides.pdf"], ["source/talk/slides.lyx"])
+scons: done building targets.
+
+*** New build: 2017-01-19 10:51:09 ***
+^^^ New build: 2017-01-19 10:51:09 ^^^
+scons: done reading SConscript files.
+scons: Building targets ...
+scons: `.' is up to date.
+scons: done building targets.
+
+*** New build: 2017-01-19 11:02:26 ***
+^^^ New build: 2017-01-19 11:02:26 ^^^
```

Comment:
*** New build: 2017-01-19 11:02:26 ***
builder_logs, et cetera
^^^ New build: 2017-01-19 11:02:26 ^^^

so if builder_logs are empty, it looks as such.

I think this is moot if we are rethinking how we log things to avoid race conditions, but just leaving this here.